package.json

```diff
@@ -47,33 +52,37 @@
   "dependencies": {
     "arrify": "^2.0.1",
     "micromatch": "^4.0.2",
-    "schema-utils": "^2.6.6"
+    "schema-utils": "^2.7.0",
+    "@types/micromatch": "^4.0.1",
+    "@types/stylelint": "^9.10.1",
+    "@types/webpack": "^4.41.17"
```

Comment:
We really need to provide all `@types` here? It can be a problem with `@types/webpack` for webpack@4 and webpack@5

Reply:
It is used in the Compiler
https://github.com/webpack-contrib/stylelint-webpack-plugin/pull/213/files#diff-e12a1b0591a9589736daa9e78e21abf6R1

I wonder if we can move to devDependecies

Reply:
Only for webpack@5, because webpack@5 contains built-in types

Reply:
Looks `micromatch` can be devDependencies only, we don't provide it in types

Reply:
I will move them

Reply:
Fixed